test/common/TestHelpers.ts

```diff
@@ -79,25 +84,31 @@ export async function bulkInsertItems(
   );
 }
 
-export async function bulkReadItems(container: Container, documents: any[], partitionKey: string) {
+export async function bulkReadItems(container: Container, documents: any[], partitionKeyProperty: string) {
   return await Promise.all(
     documents.map(async document => {
-      const options =
-        partitionKey && document.hasOwnProperty(partitionKey)
-          ? { partitionKey: document[partitionKey] }
-          : { partitionKey: {} };
+      const partitionKey = document.hasOwnProperty(partitionKeyProperty)
+        ? document[partitionKeyProperty]
+        : EMPTY_PARTITION_KEY;
```

Comment:
If this is using the `[{}]` value under the covers, then it would fail on a migrated container you ran the tests on.

Reply:
Changed to undefined